tools/automate-docs.py

```diff
@@ -0,0 +1,65 @@
+import requests
```

Comment:
Is this script running anywhere? If so, should be in a GitHub workflow, not in a docs PR

Reply:
yep, we'll need figure this out better and finalize a plan for it with Wenbing, removed it for now!